pages/itv/[processRef]/verify.tsx

```diff
@@ -51,11 +51,6 @@ export const VerifyPage: NextPage = () => {
   const tenantIds = tenants.map((tenant) => tenant.id);
 
   const idData = useDataSet(Storage.ResidentContext, "id", tenantIds);
-  const residencyData = useDataSet(
```

Comment:
All the changes in this file were needed because they were missed when the residency verification step was recently removed